test/arrays/indexing/arrLitIndexing.bad

```diff
@@ -1,4 +1 @@
-arrLitIndexing.chpl:1: warning: Anonymous associative domain literals without curly brackets are deprecated; please use curly brackets to create an associative domain of 'int(64)' indices
-arrLitIndexing.chpl:2: warning: Anonymous associative domain literals without curly brackets are deprecated; please use curly brackets to create an associative domain of 'int(64)' indices
-1 1 1
-2 2 2
+arrLitIndexing.chpl:1: error: Arrays with anonymous domains must either be defined using a list of ranges or use curly brackets.  If you were trying to create an array over a set of 'int(64)' indices, please use curly brackets.
```

Comment:
Can you add a test so that the error message here does not only occur in .bad files? I'm worried that if we fix these .bad files we'll end up with no test locking in the error.

Reply:
Added in https://github.com/chapel-lang/chapel/pull/19982/commits/fb906b524a72665948cc01f8182378762d98a6b9